src/app/tests/TestTimeSyncDataProvider.cpp

```diff
@@ -118,9 +118,9 @@ void TestTimeZoneStoreLoad(nlTestSuite * inSuite, void * inContext)
     char tzShort[]                             = "LA";
     char tzLong[]                              = "MunichOnTheLongRiverOfIsarInNiceSummerWeatherWithAugustinerBeer";
     char tzBerlin[]                            = "Berlin";
-    TimeSyncDataProvider::TimeZoneStore tzS[3] = { makeTimeZone(1, 1, tzShort, sizeof(tzShort)),
-                                                   makeTimeZone(2, 2, tzLong, sizeof(tzLong)),
-                                                   makeTimeZone(3, 3, tzBerlin, sizeof(tzBerlin)) };
+    TimeSyncDataProvider::TimeZoneStore tzS[3] = { makeTimeZone(1, 1, tzShort, sizeof(tzShort)-1),
+                                                   makeTimeZone(2, 2, tzLong, sizeof(tzLong)-1),
+                                                   makeTimeZone(3, 3, tzBerlin, sizeof(tzBerlin)-1) };
```

Comment:
The `makeTimeZone` must be replaced to use a CharSpan directly instead, which would avoid the need to use sizeofs altogether.

Reply:
Done.